### PR TITLE
Misc bug fixes

### PR DESCRIPTION
**Pull request by mmumby**
**Mine Table View**
- Commodities column on table showing false if there is a disturbance but no commodity
- Added additional check for "mine.mine_type.detail"
Before: 
Screen Shot 2020-09-18 at 12 21 44 PM

**NoW Securities**
- Added 'onConfirm' to PopConfirm to cancel editing the securities on the admin tab of a NoW

**Admin role tooltip**
- added a 'mouseLeaveDelay={0}' to remove to tooltip when user navigates away. Fixes the issue of it staying fixed on the top right corner of the screen
Screen Shot 2020-09-18 at 12 59 29 PM

**User experience UI updates**
- Removed separate table to spatial files, added them to a common "Application Files" table, updated title, added disclaimer for table
Screen Shot 2020-09-18 at 5 07 47 PM
- Updated name of "Additional Application Files" and added a disclaimer to table
Screen Shot 2020-09-18 at 5 11 20 PM

-Updated admin view to display 'contentWrappers'
Screen Shot 2020-09-18 at 5 27 03 PM
-Updated Referral/Consultation to to be displayed in 'contentWrappers'
Screen Shot 2020-09-21 at 12 41 54 PM

-Added a background when editing security info, hide edit button while in edit mode
Screen Shot 2020-09-18 at 5 27 46 PM

- Defaults to 'technical review' tab if the application has already been verified/imported and the lead inspector has been assigned
- Populate draft permit region with the region office names, default to the region the mine is located in, but allow users to change the options. 
Screen Shot 2020-09-21 at 11 19 49 AM
- Disabled tabPanes if the lead inspector has not been selected, users can only navigate to the technical review stage to review information in order to decide who the lead inspector should be 
Screen Shot 2020-09-21 at 11 46 38 AM

### CODE REVIEW

File: services/core-web/src/components/noticeOfWork/applications/NOWSubmissionDocuments.js [hunk 1]
```diff
@@ -95,29 +91,6 @@ export const NOWSubmissionDocuments = (props) => {
           }
         />
       </div>
-      <br />
-      <h4>Submission Spatial Files</h4>
```
Comment (Jsyro) [inline services/core-web/src/components/noticeOfWork/applications/NOWSubmissionDocuments.js:99]: Just curious why this is being removed.

Comment (mmumby) [reply services/core-web/src/components/noticeOfWork/applications/NOWSubmissionDocuments.js:99]: It was decided to display all the "application files" in the same table vs separating them. Having spatial files in their own table did not appear to add value. The filter was removed, added a sorter to the table to easily group files by category (there is a picture showing files together in one table if you scroll up)

File: services/core-web/src/components/Forms/permits/PermitAmendmentSecurityForm.js [hunk 2]
```diff
@@ -56,7 +57,7 @@ export const PermitAmendmentSecurityForm = (props) => (
           title="Are you sure you want to cancel?"
           okText="Yes"
           cancelText="No"
-          disabled={props.submitting}
+          onConfirm={() => props.onCancel()}
```
Comment (exzizt) [inline services/core-web/src/components/Forms/permits/PermitAmendmentSecurityForm.js:60]: We should keep the disabled line in still? That way they can't cancel out when the form is submitting

File: services/core-web/src/components/noticeOfWork/applications/administrative/NOWSecurities.js [hunk 3]
```diff
@@ -70,39 +70,49 @@ export class NOWSecurities extends Component {
...
-                  okText="Ok"
-                  cancelText="Cancel"
-                >
-                  <Button type="secondary">
+            {!this.state.isEditMode && (
+              <AuthorizationWrapper permission={Permission.EDIT_PERMITS}>
+                {isEmpty(this.props.draftAmendment) ? (
+                  <Popconfirm
+                    placement="topLeft"
+                    title="In order to edit Securities Total and a Securities Date Recieved, you need to start a Draft Permit."
```
Comment (exzizt) [inline services/core-web/src/components/noticeOfWork/applications/administrative/NOWSecurities.js:80]: Received

**Review by exzizt — DISMISSED** [review]
💯 ✅

File: services/core-web/src/components/Forms/permits/PermitAmendmentSecurityForm.js (hunk 2 above)
Comment (mmumby) [reply services/core-web/src/components/Forms/permits/PermitAmendmentSecurityForm.js:60]: Its still on the cancel button, I just removed it from the confirm actions
`<Button className="full-mobile" type="secondary" disabled={props.submitting}>
            Cancel
          </Button>`

File: services/core-web/src/components/noticeOfWork/applications/administrative/NOWSecurities.js (hunk 3 above)
Comment (mmumby) [reply services/core-web/src/components/noticeOfWork/applications/administrative/NOWSecurities.js:80]: fixed!

**Review by exzizt — APPROVED** [review]
💥 !

**Pull request merged**